hw/dv/tools/dvsim/sim.mk

```diff
@@ -120,10 +120,28 @@ cov_unr_build: gen_sv_flist
 	@echo "[make]: cov_unr_build"
 	cd ${sv_flist_gen_dir} && ${cov_unr_build_cmd} ${cov_unr_build_opts}
 
-cov_unr: cov_unr_build
+cov_unr_vcs: cov_unr_build
 	@echo "[make]: cov_unr"
 	cd ${sv_flist_gen_dir} && ${cov_unr_run_cmd} ${cov_unr_run_opts}
 
+cov_unr_xcelium: 
+ifeq ("$(wildcard $(cov_merge_db_dir))", "")
+	@echo "NO COVERAGE DATABASE DEFINED"
+else
```

Comment:
I'm not sure I'd bother with this: the rest of the file expects the command it's running to explode helpfully if some variable isn't defined. So I'd probably just put the responsibility for the check on `cov_unr_run_cmd`.